domains/transfer/Transfer.API/API/appsettings.Development.json

```diff
@@ -25,5 +25,8 @@
         "BaseUrl": "http://distribution.virk.dk",
         "User": "SomeUser",
         "Password": "SomePassword"
-    }
+    },
+    "TokenValidation:Issuer": "Us",
+    "TokenValidation:Audience": "Users",
```

Comment:
Is this good enough?

Reply:
Yes. This token is only for local development. ENV_Variables, in eo-base overwrite these when deployed to prod